libethereum/EthereumHost.h

```diff
@@ -94,6 +92,7 @@ class EthereumHost: public p2p::HostCapability<EthereumPeer>, Worker
 	std::pair<std::vector<std::shared_ptr<EthereumPeer>>, std::vector<std::shared_ptr<EthereumPeer>>> randomSelection(unsigned _percent = 25, std::function<bool(EthereumPeer*)> const& _allow = [](EthereumPeer const*){ return true; });
 	void forEachPeerPtr(std::function<void(std::shared_ptr<EthereumPeer>)> const& _f) const;
 	void forEachPeer(std::function<void(EthereumPeer*)> const& _f) const;
+	bool isSyncingInternal() const;
```

Comment:
best renamed to `isSyncing_UNSAFE` as is used in other classes (TransactionQueue/BlockQueue, iirc)